metadata-ingestion/tests/unit/utilities/test_utilities.py

```diff
@@ -1,8 +1,14 @@
 import doctest
 
+import pytest
+
 from datahub.utilities.delayed_iter import delayed_iter
 from datahub.utilities.is_pytest import is_pytest_running
-from datahub.utilities.sql_parser import SqlLineageSQLParser
+
+try:
+    from datahub.utilities.sql_parser import SqlLineageSQLParser
```

Comment:
We should remove the parser and tests as well. Make sure to mention this as breaking change in updating-datahub.

Reply:
Removed